src/services/vehicleEditor.ts

```diff
@@ -240,6 +249,14 @@ function updateVehicleSetting(args: UpdateVehicleSettingArgs): void
 			};
 			break;
 		}
+		case spinKey:
+		{
+			callback = (car): void =>
+			{
+				car.spin += value;
+			};
+			break;
+		}
```

Comment:
This case could be merged with the block used for reversed/seats/mass etc. after the changes to not use the increment value on the spinner.

Reply:
fixing